test/test-basics.js

```diff
@@ -43,20 +43,4 @@ describe('basic dag-json', () => {
     const decoded = dag.decode(dag.encode({ '/': { type: 'stringName' } }))
     same(decoded['/'].type, 'stringName')
   })
-
-  const testThrow = (fn, msg) => {
-    try {
-      fn()
-      throw new Error('Did not throw')
-    } catch (e) {
-      if (e.message !== msg) throw e
-    }
-  }
-
-  test('insufficient multibase', () => {
```

Comment:
Wouldn't it make sense to keep the test, to make sure that it isn't a silent failure that produces wrong results?

Reply:
It would need to be a different test since there’s a different error now at a very different time. But we’d be testing `js-multiformats` at that point, and we actually already have this test (base encode an encoding that doesn’t exist) in `js-multiformats` so 🤷‍♂️